### PR TITLE
[zh]translate2022-10-04-introducing-kuekue.md

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/37502

 https://kubernetes.io/blog/2022/10/04/introducing-kueue/ does not have chinese version

so need to translate it into Chinese

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+In this article, we introduce [Kueue](https://github.com/kubernetes-sigs/kueue/tree/main/docs#readme),
+an open source job queueing controller designed to manage batch jobs as a single unit.
+Kueue leaves pod-level orchestration to existing stable components of Kubernetes.
+Kueue natively supports the Kubernetes [Job](/docs/concepts/workloads/controllers/job/)
+API and offers hooks for integrating other custom-built APIs for batch jobs.
+-->
+在本文中，我们介绍了 [Kueue](https://github.com/kubernetes-sigs/kueue/tree/main/docs#readme)，
+这是一个开源作业队列控制器，旨在将批处理作业作为一个单元进行管理。
+Kueue 将 Pod 级编排留给 Kubernetes 现有的稳定组件。
+Kueue 原生支持 Kubernetes [Job](/zh/docs/concepts/workloads/controllers/job/) API，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:38]:
```suggestion
Kueue 原生支持 Kubernetes [Job](/zh-cn/docs/concepts/workloads/controllers/job/) API，
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+<!--
+Some of the most desired job queueing requirements include:
+-->
+一些最需要的作业队列要求包括：
+<!--
+- Quota and budgeting to control who can use what and up to what limit. This is not only needed in clusters with static resources like on-premises, 
+  but it is also needed in cloud environments to control spend or usage of scarce resources.
+-->
+- 用配额和预算来控制谁可以使用什么以及达到什么限制。
+这不仅在具有静态资源（如本地）的集群中需要，而且在云环境中也需要控制稀缺资源的支出或使用。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:63]:
```suggestion
  这不仅在具有静态资源（如本地）的集群中需要，而且在云环境中也需要控制稀缺资源的支出或用量。
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+<!--
+- Fair sharing of resources between tenants. To maximize the usage of available resources, any unused quota assigned to inactive tenants should be 
+  allowed to be shared fairly between active tenants.
+-->
+- 租户之间公平共享资源。
+为了最大限度地利用可用资源，应允许活动租户公平共享那些分配给非活动租户的未使用配额。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:69]:
```suggestion
  为了最大限度地利用可用资源，应允许活动租户公平共享那些分配给非活动租户的未使用配额。
```
Give 2 spaces after newline for this kind of bullets. Please change all below.

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+<!--
+Plain Kubernetes doesn't address the above requirements. In normal circumstances, once a Job is created, the job-controller instantly creates the 
+pods and kube-scheduler continuously attempts to assign the pods to nodes. At scale, this situation can work the control plane to death. There is 
+also currently no good way to control at the job level which jobs should get which resources first, and no way to express order or fair sharing. The 
+current ResourceQuota model is not a good fit for these needs because quotas are enforced on resource creation, and there is no queueing of requests. The 
+intent of ResourceQuotas is to provide a builtin reliability mechanism with policies needed by admins to protect clusters from failing over.
+-->
+普通的 Kubernetes 不能满足上述要求。
+在正常情况下，一旦创建了 Job，Job 控制器会立即创建 Pods，并且 kube-scheduler 会不断尝试将 Pod 分配给节点。
+在规模上，这种情况可能会使控制平面死机。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:90]:
```suggestion
大规模使用时，这种情况可能会使控制平面死机。
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+
+<!--
+Plain Kubernetes doesn't address the above requirements. In normal circumstances, once a Job is created, the job-controller instantly creates the 
+pods and kube-scheduler continuously attempts to assign the pods to nodes. At scale, this situation can work the control plane to death. There is 
+also currently no good way to control at the job level which jobs should get which resources first, and no way to express order or fair sharing. The 
+current ResourceQuota model is not a good fit for these needs because quotas are enforced on resource creation, and there is no queueing of requests. The 
+intent of ResourceQuotas is to provide a builtin reliability mechanism with policies needed by admins to protect clusters from failing over.
+-->
+普通的 Kubernetes 不能满足上述要求。
+在正常情况下，一旦创建了 Job，Job 控制器会立即创建 Pods，并且 kube-scheduler 会不断尝试将 Pod 分配给节点。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:89]:
```suggestion
在正常情况下，一旦创建了 Job，Job 控制器会立即创建 Pod，并且 kube-scheduler 会不断尝试将 Pod 分配给节点。
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+- 它们取代了 Kubernetes 的现有稳定组件，例如 kube-scheduler 或 Job 控制器。
+这不仅从操作的角度看是有问题的，而且重复的 Job API 也会导致生态系统的碎片化并降低可移植性。
+<!--
+- They don't integrate with autoscaling, or 
+-->
+- 它们没有集成自动扩缩容，或者
+<!--
+- They lack support for resource flexibility. 
+-->
+- 他们缺乏对资源灵活性的支持。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:112]:
```suggestion
- 它们缺乏对资源灵活性的支持。 
```

**Review by windsonsea — COMMENTED** [review]
Need to change globally: 
1. Give 2 spaces after a newline for the unordered bullets. 
2. Replace all `zh/` with `zh-cn/` in the path.

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+  labels:
+    instance-type: spot
+taints:
+- effect: NoSchedule
+  key: spot
+  value: "true"
+```
+<--
+Then you define the quotas by creating a ClusterQueue as follows:
+-->
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:253]:
```suggestion
<!--
Then you define the quotas by creating a ClusterQueue as follows:
-->
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 8]
```diff
@@ -0,0 +1,399 @@
...
+    - name: spot
+      quota:
+        min: 2000
+```
+
+<!--
+Note that the order of flavors in the ClusterQueue resources matters: Kueue will attempt to fit jobs in the available quotas according to 
+the order unless the job has an explicit affinity to specific flavors.
+-->
+注意 ClusterQueue 资源中的模板顺序很重要：Kueue 将尝试根据顺序将 Jobs 放入可用配额中，除非该 Job 与特定模板有明确的关联。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:277]:
```suggestion
注意 ClusterQueue 资源中的模板顺序很重要：Kueue 将尝试根据该顺序为 Job 分配可用配额中，除非这些 Job 与特定模板有明确的关联。
```
复数 s 可以考虑用文字表达

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 9]
```diff
@@ -0,0 +1,399 @@
...
+
+<!--
+For each namespace, you define a LocalQueue that points to the ClusterQueue above:
+-->
+对于每个命名空间，定义一个指向 ClusterQueue 的 LocalQueue：
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:282]:
```suggestion
对于每个命名空间，定义一个指向上述 ClusterQueue 的 LocalQueue：
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 10]
```diff
@@ -0,0 +1,399 @@
...
+spec:
+  clusterQueue: research-pool
+```
+
+<!--
+Admins create the above setup once. Batch users are able to find the queues they are allowed to
+submit to by listing the LocalQueues in their namespace(s). The command is similar to the following: `kubectl get -n my-namespace localqueues`
+-->
+管理员创建一次上述配置。批处理用户可以通过在他们的命名空间中列出 LocalQueues 来找到他们被允许提交的队列。
+该命令类似于以下内容：`kubectl get -n my-namespace localqueues`
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:299]:
```suggestion
该命令类似于：`kubectl get -n my-namespace localqueues`
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 11]
```diff
@@ -0,0 +1,399 @@
...
+
+<!--
+To submit work, create a Job and set the `kueue.x-k8s.io/queue-name` annotation as follows:
+-->
+要提交作业，需要创建一个 Job 并设置 `kueue.x-k8s.io/queue-name` 注释，如下所示：
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:304]:
```suggestion
要提交作业，需要创建一个 Job 并设置 `kueue.x-k8s.io/queue-name` 注解，如下所示：
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 12]
```diff
@@ -0,0 +1,399 @@
...
+Finally, if there are available empty nodes with matching node selector terms, then kube-scheduler will directly schedule the pods. If not, then
+kube-scheduler will initially mark the pods as unschedulable, which will trigger the cluster-autoscaler to provision new nodes.
+-->
+最后，如果有可用的空节点与节点选择器条件匹配，那么 kube-scheduler 将直接调度 Pod。
+如果不是，那么 kube-scheduler 将 pod 初始化标记为不可调度，这将触发 cluster-autoscaler 配置新节点。
+
+<!--
+## Future work and getting involved
+-->
+## 未来的工作和参与
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:364]:
```suggestion
## 未来工作以及如何参与
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 13]
```diff
@@ -0,0 +1,399 @@
...
+如果你在 Kubernetes 上运行批处理工作负载（需要 v1.22 或更高版本），可以尝试一下。
+我们正处于这个项目的早期阶段，正在寻求所有级别的反馈，无论是主要还是次要，所以不要犹豫，联系我们。
+我们也向其他贡献者开放，无论是修复错误或报告错误，还是帮助添加新功能或编写文档。
+你可以通过我们的[仓库](http://sigs.k8s.io/kueue)、[邮件列表](https://groups.google.com/a/kubernetes.io/g/wg-batch)或者 
+[Slack](https://kubernetes.slack.com/messages/wg-batch) 与我们联系。
+
+<!--
+Last but not least, thanks to all [our contributors](https://github.com/kubernetes-sigs/kueue/graphs/contributors) who made this project possible!
+-->
+最后很重要的一点是，感谢所有使这个项目成为可能的[我们的贡献者](https://github.com/kubernetes-sigs/kueue/graphs/contributors)！
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:399]:
```suggestion
最后是很重要的一点，感谢所有促使这个项目成为可能的[贡献者们](https://github.com/kubernetes-sigs/kueue/graphs/contributors)！
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 14]
```diff
@@ -0,0 +1,399 @@
...
+The latest [Kueue release](https://github.com/kubernetes-sigs/kueue/releases) is available on Github;
+try it out if you run batch workloads on Kubernetes (requires v1.22 or newer).
+We are in the early stages of this project and we are seeking feedback of all levels, major or minor, so please don’t hesitate to reach out. We’re 
+also open to additional contributors, whether it is to fix or report bugs, or help add new features or write documentation. You can get in touch with
+us via our [repo](http://sigs.k8s.io/kueue), [mailing list](https://groups.google.com/a/kubernetes.io/g/wg-batch) or on 
+[Slack](https://kubernetes.slack.com/messages/wg-batch).
+-->
+最新的 [Kueue 版本](https://github.com/kubernetes-sigs/kueue/releases)在 Github 上可用；
+如果你在 Kubernetes 上运行批处理工作负载（需要 v1.22 或更高版本），可以尝试一下。
+我们正处于这个项目的早期阶段，正在寻求所有级别的反馈，无论是主要还是次要，所以不要犹豫，联系我们。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:391]:
```suggestion
这个项目还处于早期阶段，我们正在搜集大大小小各个方面的反馈，请不要犹豫，快来联系我们吧！
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 15]
```diff
@@ -0,0 +1,399 @@
...
+我们也向其他贡献者开放，无论是修复错误或报告错误，还是帮助添加新功能或编写文档。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:392]:
```suggestion
无论是修复或报告错误，还是帮助添加新特性或编写文档，我们欢迎一切形式的贡献者。
```

**Comment by windsonsea** [comment]
LGTM

Preview FYI:
https://deploy-preview-37527--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/10/04/introducing-kueue/

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 16]
```diff
@@ -0,0 +1,399 @@
...
+layout: blog
+title: "Introducing Kueue"
+date: 2022-10-04
+slug: introducing-kueue
+-->
+
+<!--
+**Authors:** Abdullah Gharaibeh (Google), Aldo Culquicondor (Google)
+-->
+**作者：** Abdullah Gharaibeh（谷歌），Aldo Culquicondor（谷歌）
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:17]:
```suggestion
**作者：** Abdullah Gharaibeh（谷歌）、Aldo Culquicondor（谷歌）
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 17]
```diff
@@ -0,0 +1,399 @@
...
+
+<!--
+The latest [Kueue release](https://github.com/kubernetes-sigs/kueue/releases) is available on Github;
+try it out if you run batch workloads on Kubernetes (requires v1.22 or newer).
+We are in the early stages of this project and we are seeking feedback of all levels, major or minor, so please don’t hesitate to reach out. We’re 
+also open to additional contributors, whether it is to fix or report bugs, or help add new features or write documentation. You can get in touch with
+us via our [repo](http://sigs.k8s.io/kueue), [mailing list](https://groups.google.com/a/kubernetes.io/g/wg-batch) or on 
+[Slack](https://kubernetes.slack.com/messages/wg-batch).
+-->
+最新的 [Kueue 版本](https://github.com/kubernetes-sigs/kueue/releases)在 Github 上可用；
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:389]:
```suggestion
最新的 [Kueue 版本](https://github.com/kubernetes-sigs/kueue/releases)在 GitHub 上可用；
```

File: content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md [hunk 18]
```diff
@@ -0,0 +1,399 @@
...
+如果你在 Kubernetes 上运行批处理工作负载（需要 v1.22 或更高版本），可以尝试一下。
+这个项目还处于早期阶段，我们正在搜集大大小小各个方面的反馈，请不要犹豫，快来联系我们吧！
+无论是修复或报告错误，还是帮助添加新特性或编写文档，我们欢迎一切形式的贡献者。
+你可以通过我们的[仓库](http://sigs.k8s.io/kueue)、[邮件列表](https://groups.google.com/a/kubernetes.io/g/wg-batch)或者 
+[Slack](https://kubernetes.slack.com/messages/wg-batch) 与我们联系。
+
+<!--
+Last but not least, thanks to all [our contributors](https://github.com/kubernetes-sigs/kueue/graphs/contributors) who made this project possible!
+-->
+最后是很重要的一点，感谢所有促使这个项目成为可能的[贡献者们](https://github.com/kubernetes-sigs/kueue/graphs/contributors)！
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-10-04-introducing-kueue.md:399]:
```suggestion
最后是很重要的一点，感谢所有促成这个项目的[贡献者们](https://github.com/kubernetes-sigs/kueue/graphs/contributors)！
```

**Comment by Sea-n** [comment]
Hey @twilight0620,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by tengqm** [comment]
Overall quality is acceptable. There are nits but those are not blockers.
Considering that this is simply a blog. Kicking it in now.
/lgtm
/approve

**Pull request merged**